lib/src/main/java/com/futurewei/alcor/common/db/repo/ICacheRepository.java

```diff
@@ -28,9 +28,9 @@
 
     Map<String, T> findAllItems(Map<String, Object[]> queryParams) throws CacheException;
 
-    void addItem(T newItem) throws CacheException;
+    void addItem(T newItem) throws CacheException, Exception;
```

Comment:
@kevin-zhonghao It is generally not a good idea to change the interface because of a specific implementation.  Could you list which scenario would throw an exception rather than a cache exception?